### ISSUE TITLE
Accessibility: Focus and Voiceover for "condition" row within Hourly tables

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1010]
### Description
Within our hourly tables (on today and daily tabs), upon tabbing through the columns and values, when tabbing on the Condition row, the value is read as a "group" and the focus box remains on temperature. Is this the desired behavior?

![Image](https://github.com/weather-gov/weather.gov/assets/24654605/e9118704-e753-4dbd-81be-5d601bbfe193)

### Acceptance criteria

- [ ] Tests have been implemented or modified to capture the bug
- [ ] Changes have been tested for accessibility

**Comment by colinmurphy01** [comment]
@eric-gade when you're back, can you confirm if this is the desired behavior for the table?

**Comment by colinmurphy01** [comment]
@partly-igor do you know if this is the desired behavior?

**Comment by eric-gade** [comment]
@colinmurphy01 I am unable to replicate your problem in Safari, FF, or Chrome using normal VoiceOver navigation

**Comment by partly-igor** [comment]
There are multiple items within that cell right now which I believe it is why announces "group". You can navigate inside the group by pressing VO-down (VO = control+option). 

For the other note, I am not sure why the focus remains on the previous item before going into the group. Might just be the default behavior, but something to ask about further.

**Comment by colinmurphy01** [comment]
So this turned out to be a VO preference issue I was unaware of.

My original preferences had  `group items within` disabled

![Image](https://github.com/weather-gov/weather.gov/assets/24654605/3423b8ae-6db2-415f-adf9-0a11fa5cd3d9)

After enabling them, the screen reader and focus acted as expected

![Image](https://github.com/weather-gov/weather.gov/assets/24654605/cb33b11f-3ae5-489f-bf4b-d04dc0e59a35)
